source/SpinalHDL/Data types/bool.rst

```diff
@@ -33,14 +33,16 @@ The syntax to declare a boolean value is as follows: (everything between [] is o
      - Create a Bool assigned with ``false``
      - Bool
    * - Bool(value: Boolean)
-     - Create a Bool assigned with a Scala Boolean(true, false)
+     - Create a Bool assigned with a value from a Scala Boolean type (true,
+       false).  This implicitly converts to ``True`` or ``False`` as
```

Comment:
It does not implicitly convert since it's a Bool all along?

Reply:
This is obviously an Explicit conversion from Boolean to Bool not sure what I was thinking that day.  Maybe if we replace the word `implicit` with `explicit` does it make sense.

It starts off as a Boolean and becomes a Bool, like a regular JVM language type conversion.

Reply:
Do you think it would make more sense to describe this difference at the beginning of this chapter, a quick `Bool` vs. `Boolean`? In the sense of:
"The Bool type corresponds to a boolean value (True or False) or a single bit/wire in hardware. While named similarly it should not be confused with Scala `Boolean`s which don't describe hardware but truth values in the Scala generator code. To be used in hardware a `Boolean` must be converted to a `Bool`, i.e. using the below described `Bool` constructor."

Reply:
Definitely :)